### ISSUE TITLE
Screenshots for failed tests

### ISSUE THREAD

**Issue by mctaggartm** [reportunit/reportunit#61]
Hi, Is it possible to include screenshots for failed tests in the report?

**Comment by mr-raptor** [comment]
Hi. ReportUnit has no control about test execution, it just build report from xml.
But, I provided screenshots for failed tests in NUnit, so maybe this info will be helpful for you.
http://prnt.sc/c0z07x
Firstly, I added wrapper method for Assert, that invoke assertion and in case of fail exceptions will be processed:
http://prnt.sc/c0z0gk
I add some error info to error message (as you can see, it's html for reportUnit), also take screenshot:
http://prnt.sc/c0z0o6

In the end of test I do smth like 
`throw new AssertionException(AccumulatedErrorMessage)`
In my project I need check several errors per test, so I accumulate error messages in special object and throw it in the end. If you don't, you should throw it in ProcessException() function.

Finally that's how message looks like on ReportUnit:
http://prnt.sc/c0z19y

**Comment by kotoj** [comment]
Hi, @mr-raptor please tell if your solution working for ReportUnit v.1.5. ? I tried to do something similiar, but screenshot link is not working. It seems like ReportUnit removes html code from message, please see image below. 
![screenshotlink](https://cloud.githubusercontent.com/assets/15800766/18173335/856e59c8-7069-11e6-99e3-43fada69b371.png)

**Comment by mr-raptor** [comment]
@kotoj, yes, it works for me. Could you please provide example of your report xml?

**Comment by kotoj** [comment]
@mr-raptor  here is xml file (some data like urls, domain, machine name are deleted, but it could not influence the result). Cannot direct attache xml file: http://pastebin.com/j78ygWXS
